Tensile/Source/Client.h

```diff
@@ -291,7 +291,7 @@ void specializeData(
       if (db & 0x2) {
         std::cout << "[" << serialIdx << "] = " << val << "\n";
       }
-      initialData[serialIdx] = val++;  // actually initialize the element
+      initialData[serialIdx] = val += static_cast<DataType>(1);  // actually initialize the element
```

Comment:
In the original code, since val++ is postfix, val is incremented *after* being assigned to initialData[serialIdx].
Thus your new code is not equivalent to the original code.

Reply:
I just updated the codes.
Please review it again, thanks.